presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -210,6 +210,8 @@ public void printFinalInfo()
             reprintLine("Peak Total Memory: " + formatDataSize(bytes(stats.getPeakTotalMemoryBytes()), true));
             // Peak Task Total Memory: 1.99GB
             reprintLine("Peak Task Total Memory: " + formatDataSize(bytes(stats.getPeakTaskTotalMemoryBytes()), true));
+            // Peak Node Total Memory: 2.00GB
+            reprintLine("Peak Node Total Memory: " + formatDataSize(bytes(stats.getPeakNodeTotalMemoryBytes()), true));
```

Comment:
I don't think that we need to print all stats in the CLI - lets remove this.

Reply:
Between peak node and peak task seems like we would prefer peak node? That is how the memory limit is enforced.